packages/react/src/internal/FeatureFlags.js

```diff
@@ -17,4 +17,20 @@
  * const MyComponent = props => (<div {...props}>{aFeatureFlag ? 'foo' : 'bar'}</div>);
  */
 
-/* Currently no feature flag is in use, but keeping this file as a placeholder */
+/**
+ * Uses `value` prop to turn several components to controlled mode, notablly `<NumberInput>`.
```

Comment:
With this flag, certain components will be created in either a controlled or controlled mode based on the existence of a value prop. The following components will have the significance of their props slightly altered as outlined below.

Components: `<NumberInput>`

`value` → when provided, enables controlled mode. For the rest of the component's lifecycle, it will be controlled by this prop as it's single source of truth.
`defaultValue` → Optional starting value, used for for uncontrolled mode only (no `value` prop). The `value` prop takes precedence over `defaultValue`.
`onChange` → optional event handler. However, if `value` is provided and a handler is not, we'll throw a warning indicating the component is now read-only
`readOnly` → silences the above warning, acknowledging the read-only state of the component

Reply:
Thanks @vpicone - Put it in the code.